tests/test_cli.py

```diff
@@ -568,6 +569,13 @@ def test_bumpversion_custom_parse_semver(tmpdir):
     assert 'XXX1.1.7-master+allan2' == tmpdir.join("file15").read()
 
 
+def test_bump_version_missing_part(tmpdir):
+    tmpdir.join("file5").write("1.0.0")
+    tmpdir.chdir()
+    with pytest.raises(exceptions.InvalidVersionPartException):
```

Comment:
It would be nice to also validate the message. I don't exactly recall what `partname` would be in this case.

Reply:
Done :slightly_smiling_face: